glean-core/src/metrics/labeled.rs

```diff
@@ -33,7 +33,9 @@ lazy_static! {
     /// * `1.not_fine`
     /// * `this.$isnotfine`
     /// * `-.not_fine`
-    static ref LABEL_REGEX: Regex = Regex::new("^[a-z_][a-z0-9_-]{0,29}(\\.[a-z0-9_-]{0,29})*$").unwrap();
+    static ref LABEL_REGEX: Regex = Regex::new(
+        "^[a-z_][a-z0-9_-]{0,29}(\\.[a-z_][a-z0-9_-]{0,29})*$"
```

Comment:
We also need to change the docs [here](https://mozilla.github.io/glean/book/user/metrics/labeled_counters.html#limits), if required.

Reply:
@mdboom do the docs require updates (see the previous comment)? If so, we should also update [this page](https://mozilla.github.io/glean/book/user/metrics/labeled_strings.html).